### ISSUE TITLE
Use correct screenshot on flathub

### ISSUE THREAD

**Issue by rohmishra** [flathub/im.riot.Riot#188]
Currently the screenshot used is of a web browser, complete with browser chrome/shell visible which is not representative of the way the app looks when installed. The screenshot used should be of the app as seen when running the flatpak variation of the app.

**Comment by SISheogorath** [comment]
Agreed, these should be updated anyway, given the change in design and features over the past few months.

If anyone wants to take on that task soon, I recommend to have look at the app data standards for some detailed instructions on the aspect ration and alike: https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-dapp-screenshots

**Comment by vchernin** [comment]
I looked around on element.io, [this](https://element.io/images/homepage-all-platforms-1.png) is the only official-looking desktop screenshot I could find (from the "get element" page). Unfortunately that one isn't even remotely the right aspect ratio (16:9), so cropping might kind of ruin it. Does it make sense to take extras ourselves? (with a vanilla GTK theme probably). Maybe I'll ask around if there's anything Element recommends to use.

**Comment by SISheogorath** [comment]
I would recommend to boot a Fedora VM, install Element and make some screenshots in the right aspect ratio.

**Comment by rohmishra** [comment]
> I would recommend to boot a Fedora VM, install Element and make some screenshots in the right aspect ratio.

I'll get screenshots of the app on either a fedora VM or Gnome OS VM after work today.
